project/detector.py

```diff
@@ -0,0 +1,44 @@
+"""

```

Comment:
I think it's more common to refer to this as sentiment analysis. Maybe a more appropriate filename would be sentiment_analysis.py.

Reply:
done